tests/resources/nimbus-testbeds/vic-vsan.rb

```diff
@@ -1,3 +1,5 @@
+require ‘testframeworks/testng/testng’
```

Comment:
Stupid question: is this the correct type of quotation mark for this context?

Reply:
I've no idea - I was given it by someone from the channel. I've dropped this file entirely in favour of a simpler construction.